tests/websockets/test_graphql_transport_ws.py

```diff
@@ -33,6 +33,28 @@ async def ws_raw(http_client: HttpClient) -> AsyncGenerator[WebSocketClient, Non
     await ws.close()
     assert ws.closed
 
+    # loop cleanup code, similar to code from asyncio.run()
+    # disabled, can be enabled for debugging
+    return
+
+    loop = asyncio.get_running_loop()
```

Comment:
could we put this under a `if DEBUG_ENABLED` flag? just to make it clearer that it can be used for that?

Reply:
Sure.  The thinking here is that if we are having strange errors/warnings in the unittests, enabling this code to forcefully cleanup tasks after each test fixture should help with isolating the problem.  I can also expand on the comment.